.azure-pipelines.yml

```diff
@@ -0,0 +1,22 @@
+jobs:
+  - job: Windows
+    strategy:
+      matrix:
+        MSVC_2015:
+          MUMBLE_QT: qt5
+          MUMBLE_HOST: x86_64-pc-windows-msvc
+        MSVC_2015_NO_PCH:
+          MUMBLE_QT: qt5
+          MUMBLE_HOST: x86_64-pc-windows-msvc
+          MUMBLE_NO_PCH: 1
+    steps:
+    - script: git submodule --quiet update --init --recursive
+      displayName: 'Fetch submodules'
+    - powershell: |
+        Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
+        choco install 7zip.install -y
```

Comment:
Just thinking, is choco really the way to install deps here? Is that the easiest/recommended way [to install 7z]?

Reply:
It's what Microsoft uses for their images: https://github.com/Microsoft/azure-pipelines-image-generation/blob/b63fb395e888a0d365d5324b4f12aa9f718697e7/images/win/scripts/Installers/Install-7zip.ps1